adc/ad7779/ad7779-driver.c

```diff
@@ -36,8 +36,14 @@
 #define COL_CYAN    "\033[1;36m"
 #define COL_NORMAL  "\033[0m"
 
-#define LOG_TAG "ad7779-drv: "
-#define log_debug(fmt, ...)     do { printf(LOG_TAG fmt "\n", ##__VA_ARGS__); } while (0)
+#define LOG_TAG "ad7779: "
+
+#ifdef NDEBUG
+#define log_debug(fmt, ...)
+#else
+#define log_debug(fmt, ...)	 do { printf(LOG_TAG fmt "\n", ##__VA_ARGS__); } while (0)
```

Comment:
[nitpicking] tab instead of spaces before `do` which makes it not aligned with `do`'s below.

Not a merge blocker but if You have the strength to fix it - then do so. Otherwise LGTM.

(btw, the same has happened in ad7779.c with `log_info` and `log_error` some time ago)

Reply:
Done

Reply:
thanks for the fixes